WeeklyTask/IPriorityTask.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace WeeklyTask
+{
+    interface IPriorityTask : IRegularTask
+    {
+        public PriorityEnum Priority { get; set; }
```

Comment:
drop access modifier
```suggestion
        PriorityEnum Priority { get; set; }
```